polyjuice/polyjuice.py

```diff
@@ -3,13 +3,13 @@
 Polyjuice
 Usage:
     polyjuice.py (-h | --help)
-    polyjuice.py [-lz]  (<input_path> <output_path>) [<config_file>]
-    polyjuice.py [-lzc] [<config_file>]
+    polyjuice.py [-lm]  (<input_path> <output_path>) [<config_file>]
+    polyjuice.py [-lcm] [<config_file>]
 Options:
   -h --help                     Show this message and exit
-  -z --zip                      Archives the output folder
```

Comment:
no more zipping? is this another change that doesnt relate to the metadata downloading?